dbc2val/requirements.in

```diff
@@ -12,18 +12,18 @@
 # and create a 'requirements.txt' file with pinned versions of all
 # dependencies:
 #
-# pip-compile requirements.in
+# pip-compile --upgrade requirements.in
 #
 # If you depend on pre-releases (of e.g. kuksa-client) use
 #
-# pip-compile --pre requirements.in
+# pip-compile --pre --upgrade requirements.in
 
 python-can ~= 4.1
 pyserial ~= 3.5
 cantools ~= 38.0
 pyyaml ~= 6.0
 can-j1939 ~= 2.0
 py_expression_eval ~= 0.3
-kuksa-client >= 0.4a4
+kuksa-client ~= 0.4.0
```

Comment:
We handle versions a bit differently in our different feeders/providers. For dbc2val with the use of an `*.in* file we have absolute control as the generated `*.txt*` file give exact versions. I.e. to start using kuksa-client 0.4.1 we must manually regenerate `requirements.txt`. For other feeders/providers we just say a compatible version and do not really control if it is 0.4.0 or 0.4.1